data.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES('Agumon','2020-02-03',10.23,'true',0);
+INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES('Gabumon','2018-11-15',8,'true',2);
+INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES('Pikachu','2021-01-7',15.04,'false',1);
+INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES('Devimon','2017-05-12',11,'true',5);
+
+
```

Comment:
- [ ] You are not using Gitflow. In Gitflow you make a `development` branch from `main` branch and you make all `feature` branches from the `development` branch and merge them into your development branch not in main branch 👍 Kindly follow the Gitflow 😊

Reply:
Thank you